onnx/checker.cc

```diff
@@ -910,9 +911,14 @@ void check_model(const ModelProto& model, CheckerContext& ctx) {
   if (ctx.get_ir_version() >= 0x00000008) {
     check_model_local_functions(model, ctx, lex_ctx);
   }
+
+  if (full_check) {
+    ShapeInferenceOptions options{true, 1, true};
```

Comment:
To be consistent with previous behavior: data_prop was not enabled.
```suggestion
    ShapeInferenceOptions options{true, 1, false};
```